user-guide/Advanced_Functionality/DataMiner_Systems/Setting_up_a_new_DMS/Setting_up_a_new_DMS_in_DataMiner_Cube.md

```diff
@@ -38,4 +30,6 @@ Now all DataMiner Agents that have been added to the DMS will start the initial
 When the DMS has been synchronized, you will be able to consult information about each of the Agents in the *Status* tab of the *Agents* page in *System Center*.
 
 > [!NOTE]
-> It is also possible to migrate elements from one DMA to another within a DMS. For more information, see [Migrating elements in a DataMiner System](xref:Migrating_elements_in_a_DataMiner_System).
+>
+> - You should not set the cluster name for each DMA. This will be done automatically upon adding the DMA to the DMS.
```

Comment:
```suggestion
> - Only use the *Join cluster* button when you set up the master DMA. When you add other DMAs, the cluster name will be set automatically.
```

Note that it's no longer necessary to mention the previous name of the button as versions prior to 10.0.13 are no longer supported (we'll need to remove that obsolete info from the text too, but that doesn't necessarily have to happen with this PR – it's something we're planning to do across the user guide later).